.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do we need this artefact? Correct me if I'm wrong but my guess is that it was committed accidentally.

Reply:
I did that on purpose as this contains default formatter for `.py` files, but you might be right, that it's kinda pointless given the fact that we're using different IDEs for this project (VS Code, PyCharm, vim). Will remove it.

Reply:
Thanks!